lumicks/pylake/nb_widgets/kymotracker_widgets.py

```diff
@@ -318,7 +323,7 @@ def set_show_lines(show_lines):
                 disabled=False,
                 min=1,
                 max=10,
-                tooltip="Lines below this length are filtered out",
+                tooltip="Minimum number of frames a spot has to be detected in to be considered",
```

Comment:
It would be great if we also added this change to the docstrings of the public API.

Reply:
This is done by a separate function named `filter_lines`. I added it there.